src/SFA.DAS.ForecastingTool.Web/Infrastructure/Routing/UrlParser.cs

```diff
@@ -228,22 +229,23 @@ private void ParseStandardsFromUrl(string standardsPart, ParsedUrl result)
                     return;
                 }
 
-                if (standardCode > 0 && standardQty == 0)
+                if (standardQty == 0)
                 {
                     result.IsErrored = true;
                     result.RouteValues.Add("ErrorMessage", errorMessage);
                     result.ActionName = "TrainingCourse";
                     return;
                 }
 
-                Standard standard = null;
-                if (standardQty > 0 || standardCode > 0)
+                Apprenticeship apprenticeship = null;
+
+                if (standardQty > 0)
                 {
-                    standard = _standardsRepository.GetByCodeAsync(standardCode).Result;
-                    if (standard == null)
+                    apprenticeship = _apprenticeshipRepository.GetByCodeAsync(standardCode).Result;
+                    if (apprenticeship == null)
                     {
                         result.IsErrored = true;
-                        result.RouteValues.Add("ErrorMessage", "Number of apprentices, training standard or start date invalid");
+                        result.RouteValues.Add("ErrorMessage", "Number of apprentices, training apprenticeship or start date invalid");

```

Comment:
This error is incorrect, the only thing you know is that it doesn't exist in the list of apprenticeships from the API